apps/websockets/tests/tests_notifications/tests_follow_notifications.py

```diff
@@ -39,4 +41,12 @@ async def test_follow_valid(
         'follower_status': 'ACCEPTED',
```

Comment:
It's better approach to reuse your enums from the model and avoid hardcoded values since it will be a bit harder to refactor later on

Reply:
Yeah, took it from the request data and skipped it :)